openedx/core/djangoapps/user_api/views.py

```diff
@@ -130,6 +136,25 @@ def post(self, request):
         email = data.get('email')
         username = data.get('username')
 
+        # Begin reCAPTCHA validation #
+        recaptcha_response = data.get('g-recaptcha-response')
+        url = settings.CAPTCHA_VERIFY_URL
```

Comment:
Because it's not being use at multiple places lets please use it directly instead a making a separate `url` variable.